crates/ruma-client-api/src/filter.rs

```diff
@@ -111,6 +145,30 @@ pub struct RoomEventFilter<'a> {
     /// Defaults to `LazyLoadOptions::Disabled`.
     #[serde(flatten)]
     pub lazy_load_options: LazyLoadOptions,
+
+    /// A list of relation types to include.
+    ///
+    /// An event A is included in the filter only if there exists another event B which relates to
+    /// A with a `rel_type` which is defined in the list.
+    #[cfg(feature = "unstable-msc3440")]
+    #[serde(
+        rename = "io.element.relation_types",
+        default,
+        skip_serializing_if = "<[_]>::is_empty"
+    )]
+    pub related_by_rel_types: &'a [RelationType],
```

Comment:
Why does this have a "custom" name instead of more closely matching the spec name, i.e. `relation_types`? Question equally applies to the fields below.

Reply:
That's the difference between the unstable and stable names in the MSC. Element implemented them with those unstable names, but the stable names are the names I gave to the fields.